app/views/content_items/corporate_information_page.html.erb

```diff
@@ -56,3 +56,14 @@
     </div>
   </div>
 </div>
+<% if save_this_page_enabled? %>
```

Comment:
This too

Reply:
Done! Just as a note, the reason I left the `if save_this_page_enabled?` conditional in here is because we don't want empty divs with govuk grid layout classes in the markup when the save page box is not present.